src/app/shared/components/tooltip/tooltip.component.scss

```diff
@@ -1,15 +1,5 @@
-.c-sofie-tooltip-wrapper {
+:host {
```

Comment:
The shadow and rounded corners have been removed by this change. By reinstating them, the file would look like:
```scss
:host {
  position: absolute;
  border-radius: 8px;
  background-color: var(--background-color);
  overflow: hidden;
  box-shadow: 1px 1px 2px var(--black-color); transform: translate(0, -100%);
  z-index: 9999;
}
```

Reply:
Updated